### ISSUE TITLE
Design System Staging Review - Accessibility Feedback - Crisis Line Modal

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#71937]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- 

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Consider:** Just one thing that gives me some pause. In the crisis line modal that currently exists on VA.gov, the links for 988 and 838255 each contain additional text besides just the phone number, eg. `<a href="tel:988">Call 988 and select 1</a>`. In the new crisis line modal, each one uses the `va-telephone` component and only the number is in the link text. So where a screen reader user tabbing to the link would hear "link, Call 988 and select 1" now, with the new modal they'll just hear "link, 988." 
   
   This is more consistent with the rest of VA.gov. There are a ton of reasons why using the `va-telephone` component is better than not using it and the existing links aren't consistent with the content style guide. It's a change that makes a ton of sense.
   
   But in the particular context of a Veteran in crisis seeking immediate help, a screen reader user might benefit from getting the extra context --- 988 is the number you call (and then you have to select 1), 838255 is the number you text. If a screen reader user hears the numbers in isolation and is confused about which is which, it'll take extra action on their part to find that context in the plain text next to the link. That's something most screen reader users will be able to do with no trouble, but it's still a little extra friction in an interaction where we probably don't want any friction at all.
   
   I really don't know enough to say whether this matters or not. It probably comes down to how familiar Veterans should be with the crisis line and the ways it can be contacted (ie. when Veterans hear "988" is it reasonable to expect that they'll recognize it as a phone number?), and that's definitely not something I can assess. This might be me being needlessly cautious! But I think it's worth a conversation with someone at the crisis line.
